README.md

```diff
@@ -1,9 +1,6 @@
 # Bevy Quickstart
 
-This template is a great way to get started on a new [Bevy](https://bevyengine.org/) game -- especially for a game jam! Start with a [basic project structure](#write-your-game) and [CI / CD](#release-your-game) that can deploy to [itch.io](https://itch.io).
-
-> [!Note]
-> [Play this template on itch.io!](https://the-bevy-flock.itch.io/bevy-quickstart)
+This template is a great way to get started on a new [Bevy](https://bevyengine.org/) game -- especially for a game jam! Start with a [basic project structure](#write-your-game) and [CI / CD](#release-your-game) that can deploy to [itch.io](https://the-bevy-flock.itch.io/bevy-quickstart).
```

Comment:
I prefer the previous version. This one looks to me as if it just linked to the itch website in general.

Reply:
Yeah that's fair. I'll reword this.

Reply:
Reworded it.